menas/src/main/scala/za/co/absa/enceladus/menas/controllers/SchemaController.scala

```diff
@@ -58,12 +58,12 @@ class SchemaController @Autowired()(
                        @RequestParam name: String,
                        @RequestParam format: Optional[String]): CompletableFuture[Option[Schema]] = {
 
-    val schemaType = SchemaType.fromOptSchemaName(format)
+    val schemaType: SchemaType.Value = SchemaType.fromOptSchemaName(format)
     val (url, fileContent, mimeType) = {
       try {
         val url = new URL(remoteUrl)
         val connection = url.openConnection()
-        val mimeType = connection.getContentType
+        val mimeType = SchemaController.avscConentType // only AVSC is expected to come from the schema registry
```

Comment:
```suggestion
        val mimeType = SchemaController.avscContentType // only AVSC is expected to come from the schema registry
```

Or did you mean conent as in agreement?

Reply:
Of course not, this is a typo, thanks. Fixed.